<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="mas-PC" ignored="3" name="Default test(failed)" tests="1" failures="1" timestamp="2022-04-12T07:47:39 IST" time="0.11" errors="0">
  <testcase name="TC1" time="0.016" classname="ng.NewTest">
    <failure type="java.lang.AssertionError" message="null">
      <![CDATA[java.lang.AssertionError: null
at ng.NewTest.TC1(NewTest.java:42)
at java.util.ArrayList.forEach(Unknown Source)
... Removed 29 stack frames]]>
    </failure>
  </testcase> <!-- TC1 -->
  <testcase name="TC4" time="0.0" classname="ng.NewTest">
    <ignored/>
  </testcase> <!-- TC4 -->
  <testcase name="TC3" time="0.0" classname="ng.NewTest">
    <ignored/>
  </testcase> <!-- TC3 -->
  <testcase name="TC2" time="0.0" classname="ng.NewTest">
    <ignored/>
  </testcase> <!-- TC2 -->
</testsuite> <!-- Default test(failed) -->
